stream/src/test/java/org/apache/kafka/test/MockProcessorContext.java

```diff
@@ -147,11 +157,18 @@ public long timestamp() {
         return this.timestamp;
     }
 
-    public void node(ProcessorNode node) {
+    public void pushNode(ProcessorNode node) {
```

Comment:
Yeah, I am thinking of extracting common methods out for sharing. But I oppose to extend ProcessContextImpl because ProcessContextImpl has more dependencies.